doc/languages-frameworks/vim.section.md

```diff
@@ -28,6 +28,9 @@ vim_configurable.customize {
 ```
 
 This configuration is used when Vim is invoked with the command specified as name, in this case `vim-with-plugins`.
+You can also omit `name` to customize the system-wide Vim. See the
```

Comment:
I am not sure «system-wide Vim» is a better term than «Vim itself», unless the implication is that things won't work in `nix-shell`

Reply:
Right, I wasn't happy with that. I guess "Vim itself" is better